site/server/conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,16 @@
+# --- !Ups
```

Comment:
If we are working in a PR in isolation maybe it makes sense to squash all **our** SQL changes in a single evolution. Do you agree?